common/lib/xmodule/xmodule/contentstore/mongo.py

```diff
@@ -128,7 +128,8 @@ def find(self, location, throw_on_not_found=True, as_stream=False):
                     location, fp.displayname, fp.content_type, fp, last_modified_at=fp.uploadDate,
                     thumbnail_location=thumbnail_location,
                     import_path=getattr(fp, 'import_path', None),
-                    length=fp.length, locked=getattr(fp, 'locked', False)
+                    length=fp.length, locked=getattr(fp, 'locked', False),
+                    content_digest=getattr(fp, 'md5', None)
```

Comment:
(So that future edits like this one don't also require an edit to the last existing line.)